src/atoms/Icon/Icon.stories.tsx

```diff
@@ -5,11 +5,10 @@ import { Icon, IconProps } from './Icon';
 import { icons } from './icons';
 
 type StoryType = {
-  ariaLabel?: string;
+  role: 'decorative';
```

Comment:
I think this is supposed to be: 
```suggestion
  role: 'decorative' | 'img';
```